csdc-6/adcs-simulation/cpp/src/Simulator.cpp

```diff
@@ -49,6 +53,25 @@ timestamp Simulator::update_simulation() {
     return this->simulation_time;
 }
 
+timestamp Simulator::determine_timestep() 
+{
+    //2*max error is defined as 2*0.005 degrees = 0.01 degrees
+    if (this->variableTimestep == true){
+        float calculated_timestep = (0.01*M_PI/180)/(this->system_vals.satellite.alpha_b.cwiseAbs().maxCoeff())*1000;

```

Comment:
What are these values? What do they mean? please add them as a static const member of Simulator.cpp with an appropraite name
ie

static const float name = 0.01 * M_PI / 180